src/components/Explore/Explore.tsx

```diff
@@ -104,6 +113,19 @@ const Explore: React.FunctionComponent<{
           </p>
         </EmptyChart>
       )}
+      <DisclaimerWrapper>
+        <DisclaimerBody>
+          Last updated {lastUpdatedDateString}. Learn more about{' '}
+          <ExternalLink href="https://docs.google.com/document/d/1cd_cEpNiIl1TzUJBvw9sHLbrbUZ2qCxgN32IqVLa3Do/edit">
+            our methodology
```

Comment:
FWIW this link feels a little pointless for "explore" since the only "methodology" we're applying is a 7-day average to the raw data.  So I'd be okay with dropping it.  But 🤷